### PR TITLE
Add deploy diagram breaking change

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Add breaking change notification to Announcements page.

From https://github.com/camunda/product-hub/issues/2073#issuecomment-2541096752.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [X] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/reference/announcements/870.md [hunk 1]
```diff
@@ -90,6 +92,18 @@ Camunda 8.7 introduces a streamlined architecture, consolidating core components
 
 You can download the alpha release of the unified package from the Camunda GitHub repository, either as an executable Java application (Camunda Orchestration Core) or a Docker image.
 
+:::caution breaking change: Deploy diagram modal
```
Comment (christinaausley) [inline docs/reference/announcements/870.md:95]: Maybe out of scope for this PR, but wondering if this breaking change should be mentioned anywhere else in our SM documentation? Perhaps the linked documentation here? @conceptualshark may have stronger opinions one way or another here.

**Review by christinaausley — APPROVED** [review]

File: docs/reference/announcements/870.md (hunk 1 above)
Comment (conceptualshark) [reply docs/reference/announcements/870.md:95]: The original updates were made in https://github.com/camunda/camunda-docs/pull/4688, but I think a note that a configuration _must_ be provided to select a cluster might be in order - I can follow up with that.

Comment (mesellings) [reply docs/reference/announcements/870.md:95]: Thanks @christinaausley and @conceptualshark, I appreciate the great feedback here 🚀

**Review by nithinssabu — APPROVED** [review]
Thanks @mesellings 🚀

**Comment by mesellings** [comment]
@nithinssabu This is now published to the docs (might take 15 mins to appear/build) - thanks for your help with this 👍

**Pull request merged**